providers/network/provider/provider.go

```diff
@@ -78,15 +87,17 @@ func (s *Service) ParseCLI(req *plugin.ParseCLIReq) (*plugin.ParseCLIRes, error)
 		insecure, _ = found.RawData().Value.(bool)
 	}
 
+	conf.Options["host"] = host
+	conf.Options["port"] = fmt.Sprint(port)
+	conf.Options["type"] = "host"
+	conf.Options["path"] = url.Path
+	conf.Options["runtime"] = scheme
+	conf.Options["insecure"] = fmt.Sprintf("%t", insecure)
+	conf.Host = host
+	conf.Port = int32(port)
+
 	asset := inventory.Asset{
-		Connections: []*inventory.Config{{
```

Comment:
this looks like the reverse of parsing the cli. From parsed parameters you are converting them back to strings